sources/packages/backend/libs/integrations/src/institution-integration/ier12-integration/ier12-file-detail.ts

```diff
@@ -195,6 +197,10 @@ export class IER12FileDetail implements IER12FileLine {
     record.appendOptionalStringWithFiller(this.sabcCode, 4);
     record.appendOptionalStringWithFiller(this.institutionProgramCode, 25);
     record.append(this.programLength, 1);
+    record.appendWithStartFiller(this.currentOfferingId, 10, NUMBER_FILLER);
```

Comment:
We also have an `appendNumberWithFiller` for numbers, can you please replace it?